### PR TITLE
Task 3

### PR DESCRIPTION
**Pull request by jutglar**
thats great !

### CODE REVIEW

File: task3/Ex6.Rmd [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Exercise6 - Task4"
+output: html_document
+---
```
Comment (jutglar) [inline task3/Ex6.Rmd:4]: ok

**Review by jutglar — COMMENTED** [review]
ok

**Comment by ld162** [comment]
test

**Review by Brigggi — CHANGES_REQUESTED** [review]
functions in capitals

**Review by ks472 — CHANGES_REQUESTED** [review]
no

**Pull request closed without merging**